.devcontainer/centos8-development.Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM centos:8 as base
+
+ENV PODMAN_USERNS=keep-id
+ENV PYTHON python3
+ENV PIP pip3
+ENV PYTHONDONTWRITEBYTECODE 1
+
+ENV URL_GET_PIP "https://bootstrap.pypa.io/pip/3.6/get-pip.py"
```

Comment:
Can't we just replace this with `yum install python3-pip`?

Reply:
We have it in our other Containerfiles as well so I'm not sure, I'll look into it